transformer/catalog-transformer-geojson-queryresponse/src/test/java/ddf/catalog/transformer/queryresponse/geojson/TestGeoJsonQueryResponseTransformer.java

```diff
@@ -23,10 +23,8 @@
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
-import java.util.Date;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
+import java.text.SimpleDateFormat;
+import java.util.*;
```

Comment:
:) Wonder if we can make that part of the checkstyle so it could be caught at compile-time...